frontend/src/scenes/session-recordings/player/utils/timestamp.ts

```diff
@@ -0,0 +1,5 @@
+import { dayjs } from 'lib/dayjs'
+
+export const formattedTimestamp = (currentTimestamp: dayjs.Dayjs): string => {
```

Comment:
this isn't being called?

Reply:
Sorry, hang over from trying to share logic before I realized the two timestamps were similar but different in what they needed. Gone now